presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java

```diff
@@ -182,7 +182,8 @@ public MySqlClient(BaseJdbcConfig config, ConnectionFactory connectionFactory, T
     @Override
     protected boolean filterSchema(String schemaName)
     {
-        if (schemaName.equalsIgnoreCase("mysql")) {
+        if (schemaName.equalsIgnoreCase("mysql")
+            || schemaName.equalsIgnoreCase("sys")) {
```

Comment:
Please fix indentation as below. This is the CI failure cause. 
```suggestion
                || schemaName.equalsIgnoreCase("sys")) {
```

Code style exist here: https://github.com/prestosql/presto/blob/master/DEVELOPMENT.md#code-style